### ISSUE TITLE
Batch normalization in not bottleneck blocks in residual_units

### ISSUE THREAD

**Issue by mplanaslasa** [deepinsight/insightface#928]
What is the purpose of implementing a BatchNorm layer at the beginning of each block? 

In particular, I'm analysing residual_unit_v3 basic blocks (bottle_neck=False). 

As far as we are concerned, previous block already returns normalized layers. 

Thanks in advance
